app/services/exchange_rates/average_exchange_rates_service.rb

```diff
@@ -0,0 +1,97 @@
+module ExchangeRates
+  class AverageExchangeRatesService
```

Comment:
I was wondering if we could find a more explicit name for the name of the service.
I guess you are following the convention: for example `spot_exchange_rates_services`
What does this actually do?
create the average and upload the file?
maybe we can add a comment? :thinking: